app/controllers/admin/loans_controller.rb

```diff
@@ -160,4 +162,17 @@ def prep_questionnaire
       LoanQuestionSerializer.new(i, loan: @loan)
     end.to_json
   end
+
+  def prep_transactions
+    begin
+      Accounting::Quickbooks::AccountFetcher.new.fetch
+      Accounting::Quickbooks::TransactionFetcher.new.fetch
+    rescue
+      flash.now[:error] = 'Error connecting to quickbooks'
+    end
+
+    @transactions = ::Accounting::Transaction.where(project_id: @loan.id)
```

Comment:
Yeah, will do. I added it becase `quickbooks-ruby` did not have a base exception to catch. Ill just catch it in the fetcher, and rethrow one I can catch.